core/main/src/Core/Particle/InteractionManager.ts

```diff
@@ -42,6 +43,7 @@ export class InteractionManager {
             new Collider(container),
             new Infecter(container),
             new Linker(container),
+            new Repulser(container),
```

Comment:
The `Linker` must be the last item

Reply:
> There's something to discuss about moving particles outside the `Mover`.
> 
> The particles could be moved out of canvas without the right `outMode` handling and this could cause some issues.
> 
> Actually particles follow this path:
> 
> External Interactions (Mouse) -> Update particle (this fixes the out of canvas) -> Particle Interactions (where the new Repulser is)
> 
> The `update` is done before because links need an updated particle before setting them.
> 
> Maybe moving the `Linker` to the bottom of the array fixes that. Let me know if everything works fine or it's needed some rework

I removed the unnecessary definition and moved the Repulser before the Linker. Everything seems to work fine, the links disappear when a particle moves outside the canvas. I'm not sure what to check for, so please confirm it's working :D If it's not working, just tell me what to look for in detail :)

I'll merge the dev to this next.

Reply:
The effect looks really good, it's exactly what I wanted! Good job!